package.json

```diff
@@ -27,6 +27,7 @@
     "lint:text": "textlint 'README.md' 'docs/*.md' 'docs/**/*.md'",
     "prettier": "prettier --config .prettierrc.json --write '**/**/**/*.js' '**/**/*.js' '**/*.js' '*.js'",
     "prettier:text": "prettier 'README.md' 'docs/*.md' 'docs/**/*.md' --no-config --tab-width 4 --print-width 120 --write --prose-wrap always",
+    "start": "node ./bin/iotagent-json",
```

Comment:
New npm target should be described in the corresponding .md (not remembering which one right now... but we have a description of all the different npm targets in some place)

Reply:
Fixed 0069f53